docs/DependencyAnalysis.rst

```diff
@@ -87,17 +87,17 @@ and record it as cascading.
     ``private`` by analogy with the Swift ``private`` keyword.
 
 
-External Dependencies
+Intermodule Dependencies
 =====================
 
-External dependencies, including imported Swift module files and Clang headers,
-are tracked using a special ``depends-external`` set. The Swift driver
+Intermodule dependencies, including imported Swift module files and Clang headers,
+are tracked using a special ``depends-intermodule`` set. The Swift driver
```

Comment:
This isn't quite right either, though. This section is a dumping ground of files; it might not have anything to do with being cross-module. (A bridging header for an app target goes in this section, for example, and while that's not exactly the *same* module, it's not exactly a *different* module either.)

Reply:
I had a thought, but rather than interrupt you to try it out, I have just made the change. Can always changed it back to "external". I changed it to "extramodule". I guess "exomodule" could work, too. :)

Reply:
I just don't think "module" is the right boundary to be focusing on. Maybe just "files" or "external-files"? That doesn't say why the source files aren't included, but perhaps that's self-evident.